src/domain/whitelist/create.js

```diff
@@ -0,0 +1,12 @@
+const { Whitelist } = require('../../infra/database/models')
+
+async function create({ address, tag }) {
+    tag = tag ? tag : 'personal-invite'
+    bulkData = address.map((addr) => {
+        return { invokerAddress: addr, tag: tag }
+    })
+
+    await Whitelist.insertMany(bulkData)
```

Comment:
Please check if timestamp is also logged.

Reply:
already did. Its getting auto populated.